web2/htdocs/index.html

```diff
@@ -1929,8 +1956,11 @@ <h2>Scheduled Backups</h2>
                   <td class="retain" data-sort="[[= job.keep_n ]]" data-sort-as="number">
                     [[= pluralize(job.keep_n, 'backup') ]] / [[= pluralize(job.keep_days, 'day') ]]<br>
                   </td>
-                  <td><a href="#">edit</a> | <a href="#">delete</a> | <a href="run:[[= job.uuid ]]">run</a></td>
-                </tr>
+                  [[ if (job.paused == 0) { ]]
+                    <td> <a href="delete:[[= job.uuid ]]">delete</a> | <a href=   "run:[[= job.uuid ]]">run</a> | <a href="pause:[[= job.uuid ]]">pause</a></td>
+                     [[ } else { ]]
+                     <td> <a href="delete:[[= job.uuid ]]">delete</a> | <a href="run:[[= job.uuid ]]">run</a> | <a href="unpause:[[= job.uuid ]]">unpause</a></td>
```

Comment:
Can we avoid the duplication on the delete + run links, and just nest the pause/unpause links in the if/else bodies?